sinks/sink-parquet/src/sink.rs

```diff
@@ -152,6 +152,9 @@ impl Sink for ParquetSink {
         ctx: &Context,
         batch: &Value,
     ) -> Result<CursorAction, Self::Error> {
+        let len = batch.as_array_of_objects().unwrap().len();
+        info!(ctx = %ctx, batch = %len, "handling data");
```

Comment:
This will fail if the value returned by the transform function is null. I think we can remove this logging lines since the code in `sink-common` already logs something for each batch.

```suggestion
```